react/styles.css

```diff
@@ -11,3 +11,6 @@
   height: 10px;
   width: 10px;
 }
+
+.arrowLeft {}
```

Comment:
Is this necessary?

Reply:
Yes, we keep the CSS handles to avoid undefined classes in DOM.

Reply:
Yeah, if you don't put it will put a undefined class to the element